config/default.yml

```
@@ -415,6 +415,14 @@ Rails/HttpStatus:
     - numeric
     - symbolic
 
+Rails/I18nLazyLookup:
+  Description: 'Checks for places where I18n "lazy" lookup can be used.'
+  StyleGuide: 'https://rails.rubystyle.guide/#lazy-lookup'
```

Can you add the following reference address?

```suggestion
  StyleGuide: 'https://rails.rubystyle.guide/#lazy-lookup'
  Reference: https://guides.rubyonrails.org/i18n.html#lazy-lookup
```